packages/governance/src/breakTie.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-plusplus */
+// @ts-check
+
+/**
+ * Randomly shuffle an array
+ * https://stackoverflow.com/a/12646864
+ *
+ * @param {Position[]} array
+ */
+function shuffle(array) {
+  for (let i = array.length - 1; i > 0; i--) {
```

Comment:
```suggestion
  for (let i = array.length - 1; i > 0; i -= 1) {
```
Our style rules allow `-=` and `+=` even though they don't allow `--` and `++`.